commentHandler.go

```diff
@@ -194,15 +196,19 @@ func isValidCommand(body string, trigger string) bool {
 
 func getMaintainers(owner string, repository string) []string {
 	client := http.Client{}
-	req, _ := http.NewRequest("GET", fmt.Sprintf("https://github.com/%s/%s/raw/master/MAINTAINERS", owner, repository), nil)
+
+	maintainersFile := getEnv(maintFileEnvName, defaultMaintFile)
```

Comment:
Looks good, but what about maintainersEnv?

Reply:
instead of `maintFileEnvName`?

Reply:
Yes or maintainersFileEnv